components/script/stylesheet_loader.rs

```diff
@@ -91,6 +96,94 @@ pub struct StylesheetContext {
     resource_timing: ResourceFetchTiming,
 }
 
+impl StylesheetContext {
+    fn unminify_css(&self, data: Vec<u8>, file_url: ServoUrl) -> Vec<u8> {
+        if !self.document.root().window().unminify_css() {
+            return data;
+        }
+
+        let mut style_content = data;
+
+        // Write the minified code to a temporary file and pass its path as an argument
```

Comment:
Since a lot of this is duplicated with the script code, can we extract it into some common helpers in a new components/script/umminify.rs file?

Reply:
Yes, this was my next step

Reply:
I removed duplicate code, I tried to not break existing code, please review carefully